app/bundles/UserBundle/Config/services.php

```diff
@@ -24,4 +24,5 @@
 
     $services->alias('mautic.user.model.role', \Mautic\UserBundle\Model\RoleModel::class);
     $services->alias('mautic.user.model.user', \Mautic\UserBundle\Model\UserModel::class);
+    $services->get(\Mautic\UserBundle\Form\Validator\Constraints\NotWeakValidator::class)->tag('validator.constraint_validator');
```

Comment:
I am not seeing the service definition so I am assuming the service is auto loaded.

Are we planning on moving more services to be autoloaded for the next release? If not, I think we should define a service for a consistent code style.

Reply:
Mautic 5 is using autowiring and autoconfiguration for all services. All the services defined in config.php files should be either removed or moved to services.php to keep BC.

Reply:
Should we give it an alias as well?

Reply:
No need since it's a new service and so we don't have to keep an artificial alias for the service for BC purposes. We can use FQCN instead.